CONTRIBUTING.md

```diff
@@ -145,7 +145,7 @@ suggested modifications and push additional commits to your feature branch. Post
 a comment after pushing. New commits show up in the pull request automatically,
 but the reviewers are notified only when you comment.
 
-Pull requests must be cleanly rebased on top of master without multiple branches
+Pull requests must be cleanly rebased on top of the master without multiple branches
```

Comment:
```suggestion
Pull requests must be cleanly rebased on top of the base branch without multiple branches
```
In this context, `master` is a branch name. However, the primary branch is actually `v2` now, so I think this can be slightly more generic.